src/ape_compile/_cli.py

```diff
@@ -4,8 +4,8 @@
 
 import click
 
+from ape.options import plugin_helper
```

Comment:
Still think an `ape.cli` package containing types and helpers for building CLI extensions for ape should be exposed (different than `_cli.py` which is the actual cli endpoint for ape)

Reply:
Yes, I agree and have already implemented this but I kind of needed this logging stuff taken care first.
The `ape.cli` package will be a fast follow!

Reply:
Sounds good!

Reply:
Well, maybe I can sort of initialize that here